tests/UNMOCKED_end2end_tests/real_active_learning_run.py

```diff
@@ -118,7 +119,7 @@ def real_active_learning_run(path_to_dataset: str, token: str, dataset_id: str,
     if len(sys.argv) == 1:
         path_to_dataset = "/Users/malteebnerlightly/Documents/datasets/clothing-dataset-small-master/train"
         token = os.getenv("TOKEN")
-        dataset_id = "603606ed2176760032373065"
+        dataset_id = "603e068c23de290032d7bb55"
```

Comment:
I don't understand why you're so eager to share your personal information like the dataset id and your directory structure. You could have packed all of this in a simple script to save you the work of typing it everytime.